contracts/tokenLocking/TokenLocking.sol

```diff
@@ -112,7 +98,7 @@ contract TokenLocking is TokenLockingStorage, DSMath { // ignore-swc-123
     makeConditionalDeposit(_token, _amount, msg.sender);
 
     Lock storage lock = userLocks[_token][msg.sender];
-    emit UserTokenDeposited(_token, msg.sender, lock.balance, lock.timestamp);
+    emit UserTokenDeposited(_token, msg.sender, lock.balance);
```

Comment:
Do we have to worry about changing the event signatures here?

Reply:
I don't think it's a problem. I don't think anything is looking for / using these signatures?

Reply:
Just checking -- not ColonyJS or anything?

Reply:
I mean, they're visible there, so the definitions etc will need updating, but I don't know of anything actually consuming them - they're just there for completeness so far, as far as I know.